lightning-block-sync/src/poll.rs

```diff
@@ -140,6 +141,19 @@ impl ValidatedBlockHeader {
 
 		Ok(())
 	}
+
+    /// Returns the [`BestBlock`] corresponding to this validated block header,
+    /// which can be passed into [`ChannelManager::new`] as part of its
+    /// [`ChainParameters`]. Useful for ensuring that the [`SpvClient`] and
+    /// [`ChannelManager`] are synced to the same block during a fresh start.
+    ///
+    /// [`SpvClient`]: crate::SpvClient
+    /// [`ChainParameters`]: lightning::ln::channelmanager::ChainParameters
+    /// [`ChannelManager`]: lightning::ln::channelmanager::ChannelManager
+    /// [`ChannelManager::new`]: lightning::ln::channelmanager::ChannelManager::new
+    pub fn best_block(&self) -> BestBlock {
```

Comment:
Let's name this `to_best_block` since it is a borrowed-to-owned conversion, rather than a property accessor.

https://rust-lang.github.io/api-guidelines/naming.html#ad-hoc-conversions-follow-as_-to_-into_-conventions-c-conv